src/SFA.DAS.ApprenticeshipEarnings.MessageHandlers.UnitTests/SFA.DAS.Funding.ApprenticeshipEarnings.MessageHandlers.UnitTests.csproj

```diff
@@ -39,4 +39,8 @@
     <ProjectReference Include="..\TestHelpers\SFA.DAS.Funding.ApprenticeshipEarnings.TestHelpers.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="EventHandlers\" />
+  </ItemGroup>
+
```

Comment:
This folder doesn't exist I think, I don't think there are any tests left in this project? Can it be removed?